tests/components/ecobee/test_humidifier.py

```diff
@@ -1,130 +1,79 @@
 """The test for the ecobee thermostat humidifier module."""
-from unittest.mock import patch
+from unittest import mock
 
 import pytest
 
-from homeassistant.components.ecobee.humidifier import MODE_MANUAL, MODE_OFF
-from homeassistant.components.humidifier import DOMAIN as HUMIDIFIER_DOMAIN
+from homeassistant.components.ecobee import humidifier as ecobee
 from homeassistant.components.humidifier.const import (
-    ATTR_AVAILABLE_MODES,
-    ATTR_HUMIDITY,
-    ATTR_MAX_HUMIDITY,
-    ATTR_MIN_HUMIDITY,
     DEFAULT_MAX_HUMIDITY,
     DEFAULT_MIN_HUMIDITY,
     DEVICE_CLASS_HUMIDIFIER,
     MODE_AUTO,
-    SERVICE_SET_HUMIDITY,
-    SERVICE_SET_MODE,
     SUPPORT_MODES,
 )
-from homeassistant.const import (
-    ATTR_DEVICE_CLASS,
-    ATTR_ENTITY_ID,
-    ATTR_FRIENDLY_NAME,
-    ATTR_MODE,
-    ATTR_SUPPORTED_FEATURES,
-    SERVICE_TURN_OFF,
-    SERVICE_TURN_ON,
-    STATE_OFF,
-)
 
-from .common import setup_platform
+from .common import data_fixture, ecobee_fixture
+
+__all__ = ["data_fixture", "ecobee_fixture"]
+
+MODE_MANUAL = "manual"
+MODE_OFF = "off"
 
-DEVICE_ID = "humidifier.ecobee"
 
+@pytest.fixture(name="humidifier")
+def humidifier_fixture(data):
+    """Set up ecobee thermostat object."""
+    return ecobee.EcobeeHumidifier(data, 1)
```

Comment:
Please don't change the tests like this. They were correct before. We want to use the core interfaces to start the tests and assert core state and call services with the core service registry.

https://developers.home-assistant.io/docs/development_testing#writing-tests-for-integrations